src/scanoss/inspection/policy_check.py

```
@@ -229,21 +229,21 @@ def _get_components_from_results(self, results: Dict[str, Any]) -> list or None:
             for c in component:
                 component_id = c.get('id')
                 if not component_id:
-                    self.print_stderr(f'WARNING: Result missing id. Skipping.')
+                    self.print_debug(f'WARNING: Result missing id. Skipping.', c)
                     continue
                 status = c.get('status')
                 if not component_id:
-                    self.print_stderr(f'WARNING: Result missing status. Skipping.')
+                    self.print_debug(f'WARNING: Result missing status. Skipping.', c)
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)